abuild.in

```diff
@@ -90,7 +90,11 @@ cleanup() {
 				abuild-rmtemp "$BUILD_ROOT"
 			fi;;
 		pkgdir) msg "Cleaning up pkgdir"; rm -rf "$pkgbasedir";;
-		srcdir) msg "Cleaning up srcdir"; rm -rf "$srcdir";;
+		srcdir)
+			msg "Cleaning up srcdir";
+			options_has "chmod-clean" && test -d "$srcdir" && chmod -R +w "$srcdir"
```

Comment:
I wonder if a normal if statement would be preferred, and if we test for the existence why not include the rm in the if statement?